cg/io/txt.py

```diff
@@ -23,8 +24,10 @@ def write_txt(content: list[str] | str, file_path: Path) -> None:
 
 
 def concat_txt(
-    file_paths: list[Path], target_file: Path, str_content: Optional[List[str]] = None
-) -> None:
+    file_paths: list[Path | str],
+    target_file: Path | None = None,
+    str_content: Optional[List[str]] = None,
```

Comment:
```suggestion
    str_content: List[str] | None = None,
```
new PEP standard that we try to implement. instead of typing with `Optional` we use `| None`